qemu/tests/block_hotplug_in_pause.py

```diff
@@ -0,0 +1,177 @@
+
+    def rw_disk_in_guest(session, plug_disks, iteration):
+        """
+        Do read/write on hotplug disks
+        """
+        logging.info("Read/Writ on block device after hotplug.")
+        disk_op_timeout = int(params.get("disk_op_timeout", 360))
+        disk_letter = params.objects("disk_letter")
+        for disk in plug_disks:
+            if params.get("os_type") == "linux":
+                test_cmd = params.get("disk_op_cmd") % (disk, disk)
+            elif params.get("os_type") == "windows":
+                if iteration == 0:
+                    error_context.context("Format disk", logging.info)
+                    utils_misc.format_windows_disk(session, disk,
+                                                   mountpoint=disk_letter[int(disk)-1])
+                test_cmd = params.get("disk_op_cmd") % (disk_letter[int(disk)-1],
+                                                        disk_letter[int(disk)-1])
+                test_cmd = utils_misc.set_winutils_letter(session, test_cmd)
+            else:
+                test.cancel("Unsupported OS type '%s'" % params.get("os_type"))
+
+            status, output = session.cmd_status_output(test_cmd,
+                                                       timeout=disk_op_timeout)
+            if status:
+                test.fail("Check for block device rw failed."
+                          "Output: %s" % output)
+
+    blk_num = int(params.get("blk_num", 1))
+    repeat_times = int(params.get("repeat_times", 3))
+    get_disk_cmd = params.get("get_disk_cmd")
+    img_list = params.get("images").split()
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    for iteration in range(repeat_times):
+        device_list = []
+        if params.get("need_plug") == "yes":
+            error_context.context("Run block hotplug/unplug for iteration:"
+                                  "%d" % iteration)
+            error_context.context("Plug device", logging.info)
+            disks_before_plug = find_disk(session, get_disk_cmd)
+
+            if params.get("stop_vm_before_hotplug", "no") == "yes":
+                error_context.context("Stop VM before hotplug")
+                vm.pause()
+
+            for num in range(blk_num):
+                image_name = img_list[num + 1]
+                devs = block_hotplug(vm, image_name)
+                if devs:
+                    device_list.extend(devs)
+
+            if vm.is_paused():
+                error_context.context("Resume vm after hotplug")
+                vm.resume()
+
+            block_check_in_guest(session, disks_before_plug, blk_num, get_disk_cmd)
+
+            if params.get("disk_op_cmd"):
+                plug_disks = get_plug_unplug_disks(disks_before_plug,
+                                                   find_disk(session, get_disk_cmd))
+                rw_disk_in_guest(session, plug_disks, iteration)
```

Comment:
It is not match with steps of polarion case which did not do any operations for hotplued disks.
So update the polarion case or auto case, please. thanks.